src/Raven.Server/Documents/Indexes/Static/IndexedDocumentsEnumerator.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace Raven.Server.Documents.Indexes.Static
+{
+    public class IndexedDocumentsEnumerator : IEnumerable<DynamicDocumentObject>
+    {
+        private readonly IEnumerable<Document> _docs;
+        private readonly DynamicDocumentObject _dynamicDocument = new DynamicDocumentObject();
+
+        public IndexedDocumentsEnumerator(IEnumerable<Document> docs)
+        {
+            _docs = docs;
+        }
+
+        public IEnumerator<DynamicDocumentObject> GetEnumerator()
```

Comment:
Yes

*Hibernating Rhinos Ltd  *

Oren Eini* l CEO l *Mobile: + 972-52-548-6969

Office: +972-4-622-7811 *l *Fax: +972-153-4-622-7811



On Thu, Jun 9, 2016 at 10:05 PM, Arkadiusz Paliński <
notifications@github.com> wrote:

> In src/Raven.Server/Documents/Indexes/Static/IndexedDocumentsEnumerator.cs
> <https://github.com/ayende/ravendb/pull/3826#discussion_r66512039>:
>
> > +﻿using System.Collections;
> > +using System.Collections.Generic;
> > +
> > +namespace Raven.Server.Documents.Indexes.Static
> > +{
> > +    public class IndexedDocumentsEnumerator : IEnumerable<DynamicDocumentObject>
> > +    {
> > +        private readonly IEnumerable<Document> _docs;
> > +        private readonly DynamicDocumentObject _dynamicDocument = new DynamicDocumentObject();
> > +
> > +        public IndexedDocumentsEnumerator(IEnumerable<Document> docs)
> > +        {
> > +            _docs = docs;
> > +        }
> > +
> > +        public IEnumerator<DynamicDocumentObject> GetEnumerator()
>
> You mean to create own implementation of IEnumerator and handle MoveNext
> method, don't you?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/ayende/ravendb/pull/3826/files/b406556575bacbf98ff267c2fe34a3d2d14e68b5#r66512039>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAHIs3-4LoFJUhf9Dz_vOgNXuCQZ8coeks5qKHIdgaJpZM4IyBUw>
> .
>